package.json

```diff
@@ -106,6 +105,7 @@
     "husky": "^8.0.3",
     "jest": "^29.4.1",
     "lint-staged": "^13.1.0",
+    "lokijs": "^1.5.12",
```

Comment:
If you are sure lokijs is required when user runs cli on his computer - please remove this dependency and and install it again without -D flag, because currently it's in "devDependencies" section in package.json which means it should go to the package itself and is only used during the development process

Reply:
To use WalletConnect server side, you'll need to install the "lokijs" dependency. If you are seeing this error during a build / in an SSR environment, you can add "lokijs" as a devDependency to make this error go away.

Reply:
<img width="933" alt="image" src="https://user-images.githubusercontent.com/22689890/221797378-f80004ee-5ae7-4f14-9fa0-91b86ad5ca6d.png">
It's in wallet connect codes

Reply:
Is it better to move it from devDeps to Deps?